Source/Core/Core/PowerPC/PPCAnalyst.cpp

```diff
@@ -673,6 +673,11 @@ u32 PPCAnalyzer::Analyze(u32 address, CodeBlock *block, CodeBuffer *buffer, u32
 			num_inst++;
 			memset(&code[i], 0, sizeof(CodeOp));
 			GekkoOPInfo *opinfo = GetOpInfo(inst);
+			if (!opinfo)
+			{
+				PanicAlert("Invalid PowerPC opcode: %x.", inst.hex);
+				Crash();
```

Comment:
iirc the PanicAlert is able to crash itself, so this Crash() call isn't needed imo.